plugin-server/tests/helpers/sql.ts

```diff
@@ -335,8 +335,9 @@ export const createOrganization = async (pgClient: Pool) => {
     return organizationId
 }
 
-export const createTeam = async (pgClient: Pool, organizationId: string) => {
+export const createTeam = async (pgClient: Pool, organizationId: string, token?: string) => {
     const team = await insertRow(pgClient, 'posthog_team', {
+        id: Math.round(Math.random() * 10000000),
```

Comment:
what's the goal with this?

Reply:
I could not for the life of me figure out what racy behavior was causing tests to fail due to "team pkey already existing", even though we reset the db before each test. 

Adding this is a safe way to ensure we don't have that issue

Reply:
maybe add a comment explaining it, so we don't accidentally remove it